go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/tendermint/tm-db v0.6.7-0.20211203155021-4fa83b55a0b5
 	github.com/tharsis/ethermint v0.10.0-alpha1
 	go.opencensus.io v0.23.0
-	google.golang.org/genproto v0.0.0-20211223182754-3ac035c7e7cb
+	google.golang.org/genproto v0.0.0-20220107163113-42d7afdf6368
```

Comment:
nit, run `go mod tidy` to clean up the go.mod

Reply:
@fede  getting the following output: 
```
        github.com/go-playground/validator/v10 loaded from github.com/go-playground/validator/v10@v10.2.0,
        but go 1.16 would select v10.4.1

To upgrade to the versions selected by go 1.16:
        go mod tidy -go=1.16 && go mod tidy -go=1.17
If reproducibility with go 1.16 is not needed:
        go mod tidy -compat=1.17
```

Should I upgrade the verison?